localization/ekf_localizer/src/warning_message.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ekf_localizer/warning_message.hpp"
+
+#include <string>
+
+std::string delayTimeWarningMessage(const double delay_time)
+{
+  return fmt::format("Delay time {} cannot be less than zero.", delay_time);
```

Comment:
The error message is changed from the original one. I think it is better to leave the message as it was.

Reply:
The reason I changed the error message is because the original error message says `Pose time stamp is inappropriate` so it cannot be used for both pose and twist. 
Also, `delay_time = max(delay_time, 0);` is not included in the `delayTimeWarningMessage` function, so if we want to notify that `set delay to 0[s]`, we should add another `warning_.warnThrottle` in `ekf_localizer.cpp`.

Reply:
I got the argument about `set delay to 0[s]` , but I think we should leave the pose/twist message as it could be useful.

Reply:
Updated in [d83f293](https://github.com/autowarefoundation/autoware.universe/pull/2694/commits/d83f293dae87f1bed33d7651320be8ed1f5bc34b)